src/SWP/Bundle/ContentBundle/Controller/ArticleController.php

```diff
@@ -115,6 +119,24 @@ public function updateAction(Request $request, $id)
         return $this->handleView(View::create($form, 500));
     }
 
+    private function reactOnStatusChange($originalArticleStatus, $article)
```

Comment:
For a more general, universal solution, you could add a doctrine entity listener